results/v1.20/known-test-failures.md

```diff
@@ -0,0 +1,23 @@
+# Known Test Failures for v1.20
+
+This page aims to summarize the causes of known test failures when running
+this version of Kubernetes conformance tests against a Supervisor cluster
+running vSphere Pods.
+
+- Test: `ServiceAccounts should mount projected service account token`
+
+	Failure Reason: Bug
```

Comment:
Maybe add a bit more detail about the bug here and below, detailing what the underlying cause was probably.

Reply:
Added some more details.